internal/importers/base/importer.go

```diff
@@ -180,10 +180,18 @@ func (i *Importer) ImportPackages(packages []ImportedPackage, defaultConstraintF
 	}
 
 	for _, prj := range projects {
+		source := prj.Source
+		if len(source) > 0 {
+			isDefault, err := i.isDefaultSource(prj.Root, source)
+			if err == nil && isDefault {
```

Comment:
Let's print a warning, and ignore the imported source. Then they can review the warning and tweak if necessary.

Also add a testcase in `TestBaseImporter_ImportProjects` to verify that `WantWarning` contains a substring from our warning message and that the source was set to `""`. 

e.g.

```
Ignoring imported source https://github.com/example/foo.blorp for github.com/example/foo: <error string>
```

Reply:
Do you mean it for every time we set `Source` to `""` , or just when `isDefaultSource()` returns an error? (and it should actually never return an error, as `DeduceRootProject()` that is called earlier in `loadPackages` calls the same underlying `deduceRootPath()`)

Reply:
Sorry that wasn't clear! Yes, I was referring to when `isDefaultSource` returns an error. Even though it may never happen, I'd prefer to handle it regardless.